src/runtime/rwinsys.r

```diff
@@ -42,6 +42,7 @@ void checkpollevent(){
 
 #else					/* Graphics */
 #if NT
+int iconx(int argc, char **argv);
```

Comment:
I don't think we'll get a compiler fussy enough to complain about the difference between char ** and char*[], but why take the risk? i.e. declare it as int iconx(int argc, char *argv[]);

Reply:
Only on Windows I got this warning, and I copied the function prototype verbatim from imain.r as it is actually defined.